back-end/api/quickstart/models.py

```diff
@@ -42,20 +42,14 @@ class Visibility(models.TextChoices):
   commentLink = models.TextField(default='')
 
 class Comment(models.Model):
-  id = models.CharField(primary_key=True, max_length=LONG_CHAR_LENGTH)
+  id = models.UUIDField(primary_key=True, default=uuid.uuid4, editable=False)
```

Comment:
could we use `uuid.uuid4.hex` instead as default? the `.hex` removes all the arbitrary dashes in the UUID. but up to you; doesn't really matter

Reply:
https://stackoverflow.com/questions/48438569/returning-hex-uuid-as-default-value-for-django-model-charfield 
leaving this for reference if we want to change to hex, I'll leave as is for now since using hex actually makes the tests start failing